src/breadcrumbs/breadcrumbs.scss

```diff
@@ -18,47 +18,67 @@
   user-select: none;
 }
 
-@mixin iui-breadcrumbs-item {
+@mixin iui-breadcrumbs-item($breadcrumbs-height: $iui-component-height) {
+  --_iui-breadcrumbs-item-separator-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z' /></svg>");
+
   display: flex;
-  line-height: $iui-component-height;
-  height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
+  align-items: center;
+  line-height: $breadcrumbs-height;
+  height: $breadcrumbs-height;
```

Comment:
I'm guessing this is the part that helps with header?

But I don't see much else connection tbh. Header and breadcrumbs not only look different (apart from the separator) but also have completely different implementations, especially in React. And there is not too much code, so I think we could just repeat it in header.

Plus, these two lines don't even do anything in the base breadcrumbs.

Reply:
I've removed `$breadcrumbs-height` but may need to add it in when the header work resumes.